cpp/benchmarks/io/cudftable/cudftable_writer.cpp

```diff
@@ -0,0 +1,125 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <benchmarks/common/generate_input.hpp>
+#include <benchmarks/io/cuio_common.hpp>
+#include <benchmarks/io/nvbench_helpers.hpp>
+
+#include <cudf/io/experimental/cudftable.hpp>
+#include <cudf/utilities/default_stream.hpp>
+
+#include <nvbench/nvbench.cuh>
+
+constexpr cudf::size_type num_cols = 64;
+
+void cudftable_write_common(cudf::table_view const& view, io_type sink_type, nvbench::state& state)
+{
+  auto const data_size          = static_cast<size_t>(state.get_int64("data_size"));
+  std::size_t encoded_file_size = 0;
+
+  state.set_cuda_stream(nvbench::make_cuda_stream_view(cudf::get_default_stream().value()));
+  state.exec(nvbench::exec_tag::sync | nvbench::exec_tag::timer,
+             [&](nvbench::launch& launch, auto& timer) {
```

Comment:
Remove unused parameter to avoid potential warning.
```suggestion
             [&](nvbench::launch&, auto& timer) {
```